Rock.Messaging/MQ/NamedPipes/NamedPipeQueueConsumer.cs

```diff
@@ -60,11 +65,26 @@ public void Start(string selector)
         private void StartNewPipeServer()
         {
             _pipeServer = new NamedPipeServerStream(_pipeName, PipeDirection.In, 254, PipeTransmissionMode.Message, PipeOptions.Asynchronous);
+
+#if !NETSTANDARD1_6
             _pipeServer.BeginWaitForConnection(WaitForConnectionCallBack, null);
-        }
+            
+#elif NETSTANDARD1_6
+            Task.Run(async () =>
+            {
+                await _pipeServer.WaitForConnectionAsync();
+
+                WaitForConnectionCallBack(null);
+            });
+
 
+#endif
+
+        }
+        
         private void WaitForConnectionCallBack(IAsyncResult result)
         {
+#if !NETSTANDARD1_6
```

Comment:
There should be an equivalent try/catch looking for an ObjectDisposed exception around the await call inside the Task.Run on line 75.